twindb_backup/modifiers/keeplocal.py

```diff
@@ -0,0 +1,41 @@
+import os
+from contextlib import contextmanager
+
+from subprocess import Popen, PIPE
+
+from twindb_backup.destination.local import Local
+from twindb_backup.modifiers.base import Modifier
+from twindb_backup.util import mkdir_p
+
+
+class KeepLocal(Modifier):
+    def __init__(self, input_stream, local_path):
+        """
+        Modifier that saves a local copy of the stream in local_path file.
+
+        :param input_stream: Input stream. Must be file object
+        :param local_path: path to local file
+        """
+        super(KeepLocal, self).__init__(input_stream)
+        self.local_path = local_path
+        local_dir = os.path.dirname(self.local_path)
+        mkdir_p(local_dir)
```

Comment:
`os.makedirs()` will raise an exception if the directory exists. `mkdir_p()` is a wrapper around `os.makedirs()`